### ISSUE TITLE
[Bug] Imgur cyrillic symbols issue. Notifications don't work at all

### ISSUE THREAD

**Issue by magiasegura** [Tautulli/Tautulli#563]
Branch: dev
Commit: https://github.com/drzoidberg33/plexpy/commit/d5f4a1a48a70a9804373ed98930447e0ea124a5c
OS: Ubuntu Linux Server 14.04, Python 2.7.6
Full log: [http://pastebin.com/JDxS00uJ](url)

```
Uncaught exception: Traceback (most recent call last):
File "/opt/plexpy/plexpy/logger.py", line 221, in new_run
old_run(*args, **kwargs)
File "/usr/lib/python2.7/threading.py", line 763, in run
self.__target(*self.__args, **self.__kwargs)
File "/opt/plexpy/plexpy/notification_handler.py", line 263, in notify_timeline
notify_strings, metadata = build_notify_text(timeline=timeline_data, notify_action=notify_action)
File "/opt/plexpy/plexpy/notification_handler.py", line 573, in build_notify_text
poster_url = helpers.uploadToImgur(os.path.join(plexpy.CONFIG.CACHE_DIR, 'cache-poster.jpg'), poster_title)
File "/opt/plexpy/plexpy/helpers.py", line 550, in uploadToImgur
request = urllib2.Request('https://api.imgur.com/3/image', headers=headers, data=urllib.urlencode(data))
File "/usr/lib/python2.7/urllib.py", line 1332, in urlencode
v = quote_plus(str(v))
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-6: ordinal not in range(128)
```

When add new movie to plex, PlexPy can't upload poster to Imgur and notifications don't work at all.(Use Telegram, Slack, Pushbullet). 
I think the error occurs because the names in the library in Russian

**Comment by JonnyWong16** [comment]
Thanks for the bug report.

~~Can you post an example of one of the titles?~~ Got it.

See if it's fixed in dev.

Note: _Enable Posters in Notifications_ is now disabled by default in the notification settings.

**Comment by JonnyWong16** [comment]
Fixed in v1.3.8.
